src/Traversals/Traversals.jl

```diff
@@ -319,9 +334,13 @@ end
 function preinitfn!(s::PathState, visited)
     visited[s.exclude_vertices] .= true
     s.vertices_in_exclude = visited[s.u] | visited[s.v]
-    return !s.vertices_in_exclude
+    !s.vertices_in_exclude && return VSUCCESS
```

Comment:
I would like these to simplified where possible. That is,
```
return !s.vertices_in_exclude ? VSUCCESS : VTERMINATE
```

Where we can't simplify, the "happy path" (i.e., the one that leads to VSUCCESS) should be the normal exit.